utilities/titandb/db_iter.h

```diff
@@ -139,11 +139,11 @@ class TitanDBIterator : public Iterator {
       // corresponding blob file has already been GCed out, so we
       // cannot abort here.
       if (status_.IsCorruption()) {
-        fprintf(stderr, "key:%s GetBlobValue err:%s\n",
+        fprintf(stderr, "key:%s GetBlobValue err:%s with sequence number:%lu \n",
                 iter_->key().ToString(true).c_str(),
-                status_.ToString().c_str());
-        assert(false);
-        return false;
+                status_.ToString().c_str(),
+                options_.snapshot->GetSequenceNumber());
+        abort();
```

Comment:
Instead of abort, shall we return proper status to caller (e.g. tikv) and let it handle the error?

Reply:
The comment about `DeleteFilesInRanges` is not correct. Let's remove it.

Reply:
I think for inconsistency we'd better abort to reveal bugs.

Reply:
I think we should return Status::Corruption() to tikv on read, and tikv can decide to abort.

Reply:
Okay, seems that TiKV didn't follow the best practice of iterator that check `status()` when `valid()` is false, I will export the `status()` in `rust-rocksdb`